src/graph_gui.cpp

```diff
@@ -671,6 +700,34 @@ struct BaseGraphWindow : Window {
 		}
 	}
 
+	void OnMouseOver(Point pt, WidgetID widget) override
```

Comment:
This function could use a few comments within it to clarify what it's doing. For example, I don't know the difference between `highlight_data` and `highlight_range`. 🙂